### DISCUSSION TITLE
How do I get Quickdraw badge?

### DISCUSSION THREAD

**Discussion by AidarGG03** [community/community#102055] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get Quickdraw badge?

**Comment by IceyFL** [comment]
you have to open a issue or repository and then close it within 5 minutes

**Reply by IceyFL** [reply]
https://github.com/Thinkright20/Profile-Badges

**Comment by queenofcorgis** [comment, marked as answer]
Hi @AidarGG03 . 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
